beacon-chain/params/config.go

```diff
@@ -1,36 +1,38 @@
 // Package params defines important constants that are essential to the beacon chain.
 package params
 
-const (
-	// AttesterReward determines how much ETH attesters get for performing their duty.
-	AttesterReward = 1
+import (
+	"math/big"
+)
+
+var (
 	// CycleLength is the beacon chain cycle length in slots.
-	CycleLength = 64
+	CycleLength = uint64(64)
```

Comment:
Why change all this to `uint64`, ? You are casting them to int when you use them anyway

Reply:
We do a lot of CycleLength comparisons with the fields in crystallized and active fields which they are all `uint64` by default

Reply:
Changing this to uint64 seems unnecessary to me, you are adding a lot casting to int that was not there before across the codebase

Reply:
But there was a lot of uint64(CycleLength) before. Trade off for both case, I can switch back to int.

Reply:
if thats the case could you change just CycleLength then instead of the other params, it would reduce both uint64 and int casting then

Reply:
if that is the case then its ok to leave the default params as uint64 as it is